content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,134 @@
+                                                    sling:resourceType="core/wcm/components/container/v1/container/allowedcolors"/>
+                                            </swatches>
+                                            <image
+                                                jcr:primaryType="nt:unstructured"
+                                                jcr:title="Image"
+                                                granite:hide="${!cqDesign.backgroundImageEnabled}"
+                                                sling:resourceType="granite/ui/components/coral/foundation/container"
+                                                margin="{Boolean}true">
+                                                <items jcr:primaryType="nt:unstructured">
+                                                    <columns
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                                                        margin="{Boolean}true">
+                                                        <items jcr:primaryType="nt:unstructured">
+                                                            <column
+                                                                jcr:primaryType="nt:unstructured"
+                                                                sling:resourceType="granite/ui/components/coral/foundation/container">
+                                                                <items jcr:primaryType="nt:unstructured">
+                                                                    <file
+                                                                        jcr:primaryType="nt:unstructured"
+                                                                        sling:resourceType="cq/gui/components/authoring/dialog/fileupload"
+                                                                        allowUpload="{Boolean}false"
+                                                                        autoStart="{Boolean}false"
+                                                                        class="cq-droptarget"
+                                                                        fieldLabel="Image Asset"
+                                                                        fileNameParameter="./fileName"
+                                                                        fileReferenceParameter="./backgroundImageReference"
+                                                                        mimeTypes="[image/gif,image/jpeg,image/png,image/webp,image/tiff]"
+                                                                        multiple="{Boolean}false"
+                                                                        name="./file"

```

Comment:
I'd call it `fileReference` to be consistent with the image component.

Reply:
I'd like to keep the `background` prefix to be able to differentiate in case somebody needs a component that has both a foreground and background image. To me, `backgroundImageReference` is more descriptive as `backgroundFileReference` suggests it could be more than an image. WDYT?

Reply:
makes sense, thx for clarifying.